### ISSUE TITLE
Typo in Merge Multiple Models docs

### ISSUE THREAD

**Issue by zilch42** [MaartenGr/BERTopic#1657]
Hi Maarten, 

Thanks for the latest update, I think the latest merge_models function will be really useful for me!

I think there is a typo in the min_similarity section on the Merged Multiple Models page. Should *change* below be *chance*? (occurs twice)
https://maartengr.github.io/BERTopic/getting_started/merge/merge.html#min_similarity

"""
This (dis)similarity is can be tweaked using the min_similarity parameter. Increasing this value will increase the *change* of adding new topics. In contrast, decreasing this value will make it more strict and threfore decrease the *change* of adding new topics. The value is set to 0.7 by default, so let's see what happens if we were to increase this value to `0.9``:
"""

**Comment by MaartenGr** [comment]
Yep, that is indeed a typo! If anybody is interested, a PR would be appreciated.
